lib/matplotlib/colors.py

```diff
@@ -808,7 +799,7 @@ def _reverser(func, x):
 
     def reversed(self, name=None):
         """
-        Make a reversed instance of the Colormap.
+        Return a reversed instance of the Colormap.
```

Comment:
this is just inherited, no?

Reply:
I didn't want to write something like "the same colormap subclass". That's more confusing to inexperienced users. So in your words, I didn't bother with the duplication (it's been there and may stay here as far as I'm concerned).